contrib/ossec-testing/runtests.py

```diff
@@ -92,5 +98,9 @@ def run(self, selective_test=False):
             selective_test += '.ini'
     else:
         selective_test = False
+    shutil.copy2("./ruleset/test_rules.xml", "/var/ossec/etc/rules")
```

Comment:
We must not hardcode the Wazuh installation directory (`/var/ossec`) since we can change the location, for instance, to `/opt/wazuh`.

We should check the current installation directory via _/etc/ossec-init.conf_:
```sh
$ # cat /etc/ossec-init.conf
DIRECTORY="/var/ossec"
NAME="Wazuh"
VERSION="v3.9.1"
REVISION="3921"
DATE="Sat Jun  1 14:18:33 CEST 2019"
TYPE="server"
```

1. Check that Wazuh is installed (_/etc/ossec-init.conf_ exists).
2. Check the correct Wazuh installation:
  a) `NAME` should be "Wazuh".
  b) `VERSION` should be the same as `src/VERSION`.
  c) `REVISION` should be the same as `src/REVISION`.
  d) `TYPE` must be either "server", "local" or "hybrid".
3. Get `DIRECTORY` as the current installation folder.